packages/expo-image/src/Image.types.ts

```diff
@@ -108,6 +108,12 @@ export type ImageProps = AccessibilityProps & {
    */
   cachePolicy?: ImageCachePolicy | null;
 
+  /**
+   * Determines whether to choose image source based on container size only on mount or on every resize.
+   * @platform web
+   */
+  webResponsivePolicy?: 'live' | 'initial';
```

Comment:
I would remove `web`, I don't know about any platform-specific prop that we prefix with `ios` or `android` 🤔 

```suggestion
  responsivePolicy?: 'live' | 'initial';
```